.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 11,
+    "sourceType": "module"
+  },
+  "plugins": [
+    "react"
+  ],
+  "rules": {
+    "indent": ["error", 2, { "SwitchCase": 1 }],
+    "linebreak-style": ["error", "unix"],
+    "max-len": ["warn", { "code": 100 }],
+    "quotes": ["error", "single"],
+    "react/prop-types": "off", // We should consider setting this to "warn"
```

Comment:
I'm a big fan of using PropTypes. We should consider it. :)

Reply:
Yes, I like prop-types as well, its simple to use, easy to understand, and an clean way to add defaults